### ISSUE TITLE
FTB Client - McJtyLib is not a mod (?)

### ISSUE THREAD

**Issue by lordover123** [McJty/McJtyLib#23]
The log report: http://paste.feed-the-beast.com/view/b9fd3d9a
The specific line of text:  [12:48:25] [Client thread/INFO] [FML]: FML has found a non-mod file mcjtylib-1.10-2.2.0.jar in your mods directory. It will now be injected into your classpath. This could severe stability issues, it should be removed if possible.

That's it really... I don't know what is causing this issue. When I remove McJtyLib it says it needs it to run. http://prntscr.com/d83j9g
Here's the file I get when I click the fml-client-latest.log button:
[fml-client-latest.log.zip](https://github.com/McJty/McJtyLib/files/595528/fml-client-latest.log.zip)

**Comment by McJty** [comment]
That's not a problem. That's normal. You can just ignore that line

**Comment by lordover123** [comment]
To be honest, I now feel like an idiot. I forgot to mention that my game was crashing. I was suspecting it was your mod because of that line. By any chance do you know what might be causing that from the reports I supplied? Curse, FTB, and vanilla minecraft fail but give different reasons for each of them. 

Sorry about that

**Comment by McJty** [comment]
There is no crash in that log. You'll need to give the full server log

**Comment by lordover123** [comment]
How would I go about getting that? I did just realize that in the initial post I included a screencap of Minecraft still open so it wasn't technically crashing, but I couldn't get any further and regarded it as such. Curse and the vanilla launcher specifically were the ones crashing (Curse -> Failed to create Java Virtual Machine, Vanilla wouldn't give me logs that I could find, I'll look again) 

Sorry for pulling you back to this post after it being closed, and also sorry for the late response. I try not to ghost people, but I forgot this time
